linetools/spectra/io.py

```diff
@@ -458,11 +476,15 @@ def parse_FITS_binary_table(hdulist, exten=None):
         return
     co_tags = ['CONT', 'CO', 'CONTINUUM', 'co', 'cont']
     co, co_tag = get_table_column(co_tags, hdulist, idx=exten)
+    # Finish
     xspec1d = XSpectrum1D.from_tuple((give_wv_units(wave), fx, sig, co))
+
+    if 'METADATA' in hdulist[0].header:
+        xspec1d.meta.update(json.loads(hdulist[0].header['METADATA']))
     return xspec1d
 
 def parse_linetools_spectrum_format(hdulist):
-    """ Parse a linetools-format spectrum from an hdulist
+    """ Parse an old linetools-format spectrum from an hdulist
```

Comment:
Why is this now an old format?  This format is fine for 2d flux/sig/wavelength arrays, we can simply save 2d arrays in each extension instead of 1d arrays.